lib/inferno/dsl/fhir_evaluator/ig.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
```

Comment:
From discussion - delete this file; it will be added back in FI-3440 as noted. I copied Steve's comment into that issue on Jira

Reply:
Deleted.